static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -84,6 +86,26 @@ define([
         var ophanUrl = 'https://dashboard.ophan.co.uk/graph/breakdown?ab=' + this.config.test.id;
         this.getElem('ophan').innerHTML = '<a href="' + ophanUrl + '"">graph</a>';
 
+        if (window.abConfigSwitches.abShowHypothesesDashboard) {
```

Comment:
I'd be tempted to just deploy and have it on all the time, especially as it's not user facing